### PR TITLE
update Zone and Rules

### PR DESCRIPTION
**Pull request by PlasGen**

### CODE REVIEW

File: index.org [hunk 1]
```diff
@@ -2136,6 +2136,55 @@ Modding documentation for Mindustry.
 
 * Game
 ** Rules
+    Rules is applied to map,it can be edited at map editor,you can edit your mod-added campaign map's rule through these fields.
+    
+    | field                    | type          | default    | description |
```
Comment (SimonWoodburyForget) [inline index.org]: Column `description` should be named `notes`

File: index.org [hunk 2]
```diff
@@ -1788,7 +1788,75 @@ Modding documentation for Mindustry.
...
+    | rules              | [[Rules]]       |            |
+    | requirements       | [[Objective]]   |            |
+    | configureObjective | [[Objective]]   |            |
+    | defaultStartingItems | [[ItemStack]] |            |
+    
+*** Loadout
+    Loadout is the blueprint when your core landed.
+    
+    In Loadout:
+    | '>' is stands for conveyor face to right |
```
Comment (SimonWoodburyForget) [inline index.org]: I'm having a hard time understand what this table is supposed to tell the user:

* give it a header and columns.
* the wording `x is stands for y` doesn't really make sense.
* `'>'` should be in a code block, not in single quotes.

Despite that I'm not sure I understand exactly what it's saying, I'll have to take a look at the source. I know you're defining characters to be used in relation to a block, but I'm not entirely sure why you're doing it.

File: index.org [hunk 3]
```diff
@@ -1788,7 +1788,75 @@ Modding documentation for Mindustry.
...
+    You must set a core in your custom Loadout
+    
+    Built-in loadout:
+    * ~basicShard~
+        ("  ###  ",
+        "  #1#  ",
+        "  ###  ",
+        "  ^ ^  ",
+        " ## ## ",
+        " C# C# ")
```
Comment (SimonWoodburyForget) [inline index.org]: This should be inside a code block `#+BEGIN_SRC fundamental` and `#+END_SRC` to maintain mono font spacing. I don't understand what it's supposed to be either, `(` isn't used at all in the context of JSON.

File: index.org [hunk 4]
```diff
@@ -1788,7 +1788,75 @@ Modding documentation for Mindustry.
...
+    | '^' is stands for conveyor face to up |
+    | '<' is stands for conveyor face to left |
+    | 'v' is stands for conveyor face to down |
+    | '1' is stands for coreShard |
+    | '2' is stands for coreFoundation |
+    | '3' is stands for coreNucleus |
+    | 'C' is stands for mechicalDrill with four copper ore |
+    
+    Note:
+    You must set a core in your custom Loadout
```
Comment (SimonWoodburyForget) [inline index.org]: This should be in an unordered list.

```
notes
* You must...
```

File: index.org [hunk 5]
```diff
@@ -1788,7 +1788,75 @@ Modding documentation for Mindustry.
...
+    
+    Built-in loadout:
+    * ~basicShard~
+        ("  ###  ",
```
Comment (SimonWoodburyForget) [inline index.org]: You haven't defined what `#` and ` ` are supposed to meant.

File: index.org [hunk 6]
```diff
@@ -1856,7 +1856,34 @@ Modding documentation for Mindustry.
...
+    
+    | field        | type           | default | notes |
+    |--------------+----------------+---------+-------|
+    | map          | Map            |         | this is set in map editor |
+    | mapName      | String         |         | |
+    | decorations  | [[Decoration]] |         | |
+    | enemySpawns  | int            | -1      | The amount of final enemy spawns used. -1 to use everything in the map.This amount of enemy spawns is selected randomly from the map. |
+    | distortFloor | boolean        | false   | Whether floor is distorted along with blocks. |
+    
+***** Decoration
```
Comment (SimonWoodburyForget) [inline index.org]: Why is this inside of `MapGenerator`?

File: index.org [hunk 7]
```diff
@@ -1856,7 +1856,34 @@ Modding documentation for Mindustry.
...
+    Decoration is a random generator to generate some block in floor you set.
+    
+    | field   | type      |
+    |---------+-----------|
+    | floor   | [[Block]] |
+    | wall    | [[Block]] |
+    | changce | double    |
```
Comment (SimonWoodburyForget) [inline index.org]: typo

**Comment by PlasGen** [comment]
i fixed these problems

**Pull request merged**